BooruSharp/Booru/ABooru.cs

```diff
@@ -222,23 +218,23 @@ private async Task<string> GetRandomIdAsync(string tags)
 
         private string CreateUrl(string url, params string[] args)
         {
-            if (_format == UrlFormat.indexPhp)
-                return (url + "&" + string.Join("&", args));
-            else
-                return (url + "?" + string.Join("&", args));
+            return _format == UrlFormat.indexPhp
+                ? url + "&" + string.Join("&", args)
+                : url + "?" + string.Join("&", args);
         }
 
         private string TagsToString(string[] tags)
         {
-            return tags == null ? "" : "tags=" + string.Join("+", tags.Select(x => Uri.EscapeDataString(x))).ToLower();
+            return tags != null
+                ? "tags=" + string.Join("+", tags.Select(Uri.EscapeDataString)).ToLower()
+                : "";
```

Comment:
This should probably also check for `tags.Length > 0`.

Reply:
Since it's called internally I'm not sure checking the length is really important
(And iirc the public functions already remove empty tags)